src/components/ToggleSwitch/index.tsx

```diff
@@ -33,50 +33,39 @@ const styles = StyleSheet.create({
     alignItems: 'center'
   },
   textOn: {
-    fontFamily: 'Inter',
-    fontWeight: '400',
+    fontFamily: fonts.inter[400],
     fontSize: normalizeFontSize(12),
     lineHeight: dimen.normalizeDimen(18),
     textAlign: 'center',
     textAlignVertical: 'center',
-    color: COLORS.whiteSmoke,
+    color: COLORS.almostBlackSmoke,
     position: 'absolute',
     top: dimen.normalizeDimen(1),
     left: dimen.normalizeDimen(2),
     width: dimen.normalizeDimen(22),
     height: dimen.normalizeDimen(18)
   },
   textOff: {
-    fontFamily: 'Inter',
-    fontWeight: '400',
+    fontFamily: fonts.inter[400],
     fontSize: normalizeFontSize(12),
     lineHeight: dimen.normalizeDimen(18),
     textAlign: 'center',
     textAlignVertical: 'center',
-    color: COLORS.whiteSmoke,
+    color: COLORS.almostBlackSmoke,
     position: 'absolute',
     top: dimen.normalizeDimen(1),
-    right: dimen.normalizeDimen(2),
+    right: dimen.normalizeDimen(1),
     width: dimen.normalizeDimen(22),
     height: dimen.normalizeDimen(18)
   },
   mainStyes: {
     position: 'relative',
     borderRadius: dimen.normalizeDimen(12),
+    borderWidth: 1,
+    borderColor: COLORS.gray200,
     backgroundColor: COLORS.almostBlack,
-    height: dimen.normalizeDimen(20),
-    width: dimen.normalizeDimen(42),
-    ...Platform.select({
-      ios: {
-        shadowColor: COLORS.gray200,
-        shadowOffset: {width: 0, height: 0},
-        shadowOpacity: dimen.normalizeDimen(1),
-        shadowRadius: dimen.normalizeDimen(1)
-      },
-      android: {
-        elevation: dimen.normalizeDimen(1)
-      }
-    })
+    height: dimen.normalizeDimen(22),
+    width: dimen.normalizeDimen(42)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `mainStyles` object has been modified. The `borderWidth` and `borderColor` properties have been added, and the `height` property has been increased. The platform-specific shadow styles for iOS and Android have been removed. If these changes were intentional and tested on both platforms, then it's fine. However, if they were removed unintentionally, you might want to add them back to ensure consistent styling across different platforms.

```diff
-    height: dimen.normalizeDimen(20),
+    height: dimen.normalizeDimen(22),
+    borderWidth: 1,
+    borderColor: COLORS.gray200,
-    ...Platform.select({
-      ios: {
-        shadowColor: COLORS.gray200,
-        shadowOffset: {width: 0, height: 0},
-        shadowOpacity: dimen.normalizeDimen(1),
-        shadowRadius: dimen.normalizeDimen(1)
-      },
-      android: {
-        elevation: dimen.normalizeDimen(1)
-      }
-    })
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->